### PR TITLE
Add Math documentation and pw2ceil test

### PR DESCRIPTION
**Pull request by Zavydiel**

### CODE REVIEW

File: src/math/math.cr [hunk 1]
```diff
@@ -10,20 +10,24 @@ module Math
...
     end
 
+    # ditto
     def {{name.id}}(value)
       {{name.id}}(value.to_f)
     end
   {% end %}
 
   {% for name in %w(besselj0 besselj1 bessely0 bessely1) %}
+    # Calculates the {name.id} function of *value*
```
Comment (asterite) [inline src/math/math.cr]: Should be `{{name.id}}`

**Comment by asterite** [comment]
@Zavydiel Awesome, thank you! Just made you a small correction request, then I'll merge.

**Comment by Zavydiel** [comment]
Fixed it.

**Comment by asterite** [comment]
Thanks!

**Pull request merged**